index.js

```diff
@@ -4,8 +4,8 @@
 const args = require('get-them-args')(process.argv.slice(2))
 const Haikunator = require('haikunator')
 
-const adjectives = ['red', 'orange', 'yellow', 'green', 'blue', 'indigo', 'violet']
-const nouns = ['apple', 'avocado', 'lemon', 'lime', 'orange', 'watermelon']
+const adjectives = ['red', 'orange', 'yellow', 'green', 'blue', 'indigo', 'violet', 'clean', 'safe', 'respectful', 'punctual', 'enjoyable', 'favorite']
+const nouns = ['apple', 'avocado', 'lemon', 'lime', 'orange', 'watermelon', 'carpool', 'cancellation', 'detour', 'no-show', 'rider', 'driver', 'waypoint', 'satisfaction', 'reliability', 'visibility', 'impact', 'favorite', 'itinerary', 'timeline', 'reimbursement', 'credit']
```

Comment:
```suggestion
const nouns = ['apple', 'avocado', 'lemon', 'lime', 'orange', 'watermelon', 'carpool', 'cancellation', 'detour', 'noshow', 'rider', 'driver', 'waypoint', 'satisfaction', 'reliability', 'visibility', 'impact', 'favorite', 'itinerary', 'timeline', 'reimbursement', 'credit']
```

Reply:
Love the words!

The haiku version form uses '-' as the delimiter. We might want to avoid using it in individual words, just in case someone is parsing it.